package.json

```diff
@@ -10,6 +10,9 @@
     "prepush": "make verify -j3",
     "commitmsg": "node_modules/.bin/secret-squirrel-commitmsg"
   },
+  "engines": {
```

Comment:
This is a component so not sure this is necessary. If it's a requirement can we set it to >=6.11.1 becuase I think some of our lambdas might use it and they don't support anything above 6.x.x yet.